.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+env:
+  node_version: 20
+  pnpm_version: 8
```

Comment:
current pnpm major version is 9
You can actually get it to work with the one svelte library that doesn't like pnpm 9 yet by running `pnpm i --force`.
It's up to you how you want to handle that.

Reply:
I think we should let `pnpm@9` cook for a little bit longer before we start using it

Reply:
I've updated to 9. Looks like everything works. In case anything breaks we can always choose to revert. Thanks for letting me know.